rollup/defs.bzl

```diff
@@ -1,9 +1,200 @@
-"wrapper macro for rollup_bundle rule"
+"wrapper macro for rollup rule"
 
-load("//rollup/private:rollup_bundle.bzl", rollup_lib = "lib")
+load("@aspect_bazel_lib//lib:directory_path.bzl", "directory_path")
+load("@aspect_rules_js//js:defs.bzl", "js_binary")
+load("//rollup/private:rollup.bzl", rollup_lib = "lib")
 
-rollup_bundle = rule(
```

Comment:
Why the rename? Doesn't it only produce the bundle and not do other things like a devserver or any other functionality it has?

Reply:
We've talked about this before so its better to lump breaking changes in together.
`rollup_bundle` isn't consistent with `esbuild` since we don't call that one `esbuild_bundle`. I also renamed `terser_minified` to just `terser` for consistency.

Reply:
Lets put that in a BREAKING CHANGE in the commit message?